.github/workflows/flaskproject.yaml

```diff
@@ -94,20 +94,19 @@ jobs:
           name: codeql
           path: ./${{env.PROJECT}}
 
-      - name: (REMOVE LATER) CI debugging
-        run: ls
-
-      - name: (REMOVE LATER) CORRECT FILE debugging
-        run: cat ./${{env.PROJECT}}/${{env.PROJECT}}-semgrep.sarif
-        working-directory: ./
-
+      # TODO: figure out why its name is python for some reason
+      # This is just a very temporary fix because it's just taking the python and
+      # directing it to a new file
       - name: (REMOVE LATER) CORRECT FILE debugging
-        run: cat ./${{env.PROJECT}}/python.sarif > ./${{env.PROJECT}}/${{env.PROJECT}}-codeql.sarif # TODO: figure out why its name is python for some reason
+        run: cat ./${{env.PROJECT}}/python.sarif > ./${{env.PROJECT}}/${{env.PROJECT}}-codeql.sarif 
         working-directory: ./
+      
+      - name: Set up aggregation script
+        run: chmod +x ./utils/agg.sh
+        working-directory: ./ 
         
-      # Tfw we don't rlly need docker for this
       - name: Aggregate
-        run: utils/agg.sh ${{env.PROJECT}} sastall.sarif
+        run: ./utils/agg.sh ${{env.PROJECT}} sastall.sarif
         working-directory: ./
```

Comment:
Would it be better to specify `working-directory: '.'` instead of `working-directory: ./`